Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
+    "\n",
+    "As discussed earlier in the Simulation section user can take advantage of pre-written functions and libraries to calculate and report drag force coefficient values. The \"forceCoeffs\" function, added to `controlDict` dictionary file located in `\\system` directory, takes \"cylinder\" walls as the patch and calculates the forces on in the defined directions. This was completely discussed in the Simulation section. Table 1 provides a summary of comparison between the present study and literature calculated drag coefficients values:\n",
+    "\n",
+    "| Tip Speed Ratio #  | $C_D$ Present Study| $C_D$ D. Stojković *et al.* (2002) | Relative Error [%]|\n",
+    "| :--------------------------------:|:------------------:|:--------------------------------:|:-----------------:|\n",
+    "| 0.5        | 1.2819      | 1.2740        |0.6|\n",
+    "| 1          | 1.1112      | 1.1080        |0.2|\n",
+    "| 1.5        | 0.8358      | 0.8180        |2.1|\n",
+    "| 2          | 0.5415      | 0.4611        |17.4|\n",
+    "Table 1. Comparison between the numerically predicted drag coefficient on a rotating cylinder with Re=100 and different Tip Speed Ratios.\n",
+    "\n",
+    "As shown in the Table 1 the values of the relative error percentages are below 2.1% for case studies with relatively low TSR values (below 1.5). Furthermore, Table 1 reveals that for TSR value of 2 and above the relative error percentage of corresponding drag coefficient values increases significantly. The reason behind this is the increase of the total velocity of the flow (algebraic summation of streamwise and rotating velocity components) and therefore more complex physics of the laminar boundary layer. As long as the flow regime is laminar this error can be reduced by decreasing the time step value of the simulation from 0.01 second. However, if increasing the TSR value lead into change of a flow regime from Laminar to Transitional or Turbulent flow, adequate mesh resolution and if required numerical scheme should be used to accurately estimated the drag coefficient of the rotating cylinder and simulate the flow field around and in the wake of the cylinder.\n",
+    "\n",
```

Comment:
I added the lift coefficient and some sentences are changed a bit. At the end I have also added some additional sentenced regarding the Cl results. Please check them out.